csrc/kernels/flash_api.cu

```diff
@@ -71,7 +72,13 @@ extern "C" void run_mha(
     int window_size_left,
     int window_size_right,
     float softcap,
+<<<<<<< HEAD
```

Comment:
this file has many unresolved merge issues

Reply:
it didn't save when I changed these